src/System.Devices.Gpio/System.Devices.Gpio.Samples/Program.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System.Diagnostics;
```

Comment:
All files need this at the top


// Copyright (c) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.